### ISSUE TITLE
Link to API documentation in HTTP Server root is broken

### ISSUE THREAD

**Issue by schlamar** [0xERR0R/blocky#806]
The link to the API in the HTTP server root (Swagger Rest API Documentation (Online @GitHub)) is pointing to https://htmlpreview.github.io/?https://github.com/0xERR0R/blocky/blob/0.20/docs/swagger.html which returns a 404.

**Comment by 0xERR0R** [comment]
It should be `https://htmlpreview.github.io/?https://github.com/0xERR0R/blocky/blob/v0.20/docs/swagger.html`

**Comment by 0xERR0R** [comment]
Tested with following config:

```yaml
upstream:
  default:
    - 1.1.1.1
httpPort: 4000 
```

```bash
> docker run --rm -v $(pwd)/config.yml:/app/config.yml -p 4000:4000 spx01/blocky:v0.20

> curl --silent localhost:4000 | grep swagger
        <li><a href="https://htmlpreview.github.io/?https://github.com/0xERR0R/blocky/blob/v0.20/docs/swagger.html">Swagger Rest API Documentation (Online @GitHub)</a></li>
```

Link can be opened.

**Comment by 0xERR0R** [comment]
Tested with released binary, it uses 0.20 as version number (instead of v0.20)

**Comment by 0xERR0R** [comment]
Cause: Goreleaser removes the v prefix: https://goreleaser.com/customization/templates/#fn:version-prefix
